### DISCUSSION TITLE
Null annotations

### DISCUSSION THREAD

**Discussion by jaimeebull** [SkalskiP/make-sense#228] [Q&A]
Is there a method to create null annotations on makesense?

**Comment by SkalskiP** [comment]
Hi, @jaimeebull 👋 ! What do you mean by null annotations?

**Reply by rasyidf** [reply]
Let me guess, the negative cases? where no object should be detected in the image

**Reply by SkalskiP** [reply]
Oh in that case currently we filter out all images that do not contain annotations. In theory, we could add some toggle buttons to enable/disable that functionality. Any thoughts?

**Reply by rasyidf** [reply]
Yes, that's what i thought last issues about delete image means. a toggle to enable / disable filtering of zero annotation

**Reply by SkalskiP** [reply]
I think it is a different use case. https://github.com/SkalskiP/make-sense/issues/77 is about removing the actual image - I don't want to have that image in the dataset. Here we are talking about keeping images, but without annotations.

**Reply by jonarri** [reply]
I was loking for this too, it would be very useful!! Are there any plans to implement it?

**Reply by SkalskiP** [reply]
To be quite honest, I stopped developing MakeSense quite a while ago. At the moment, I'm not adding any new features. Sorry. :)

**Reply by jonarri** [reply]
Oh, sorry to hear. Don't worry, I will find a scrappy fix to make it work ;)
Hope everything is going great in whatever you are working on now!!

**Reply by SkalskiP** [reply]
Everything is fine. Other obligations simply prevent me from further developing this project. :/
